components/nds-page-header/__tests__/nds-page-header.test.js

```diff
@@ -41,4 +41,11 @@ describe("PageHeader", () => {
 		);
 		expect(wrapper.find("div").props()["data-track"]).toEqual(false);
 	});
+
+	it("should merge classes with the container", () => {
+		const wrapper = shallow(
+			<PageHeader heading="This is our heading" className="mt--0" />
```

Comment:
So this component was already merging class? Can only see a test in the PR and no code change....

Reply:
It certainly was